src/Normalizer/Normalizer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Http\Client\Plugin\Normalizer;
+
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Normalize a request or a response into a string or an array
```

Comment:
It is...for now. We should replace it in the future with some kind of Formatter interface with different kind of formatters.